urlwatch

```diff
@@ -109,11 +111,11 @@ class NullHandler(logging.Handler):
 
 log.addHandler(NullHandler())
 
-ERROR_MESSAGE_URLS_TXT = """
-Error: You need to create a urls.txt file first.
+ERROR_MESSAGE_URLS = """
+Error: You need to create either a urls.yaml or a urls.txt file first.
```

Comment:
Or we only have `urls.yaml` and we provide a way to migrate any old `urls.txt` to `urls.yaml` on first run (if `urls.yaml` doesn't exist yet) so that there's no confusion on which format to use.